### DISCUSSION TITLE
Why does Renovate try to escape image replacements for k8s.gcr.io?

### DISCUSSION THREAD

**Discussion by uhthomas** [renovatebot/renovate#22509] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I'm using a regex manager to match docker images in `.cue` files rather than `.yaml`.

https://github.com/uhthomas/automata/blob/03c944481859a5967c35368c76cff48a498e9dc5/renovate.json#L17-L29

The PRs opened want to replace `k8s.gcr.io` with `registry\.k8s\.io`. The backslashes shouldn't be there and cause the builds to fail.

https://github.com/uhthomas/automata/pull/153

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by HonkingGoose** [comment]
Here's a minimal reproduction for us in the `renovate-reproductions` organization: https://github.com/renovate-reproductions/22509

I'll let the maintainers figure out what's going wrong here. 😄

**Comment by rarkins** [comment]
Relevant replacement code: https://github.com/renovatebot/renovate/blame/2b0a609e2652c8955405ab3613cdc2ab0b135ca5/lib/config/presets/internal/replacements.ts#L561-L572

Author @fgreinacher in https://github.com/renovatebot/renovate/pull/21103, approved by @viceice. The PR was marked as code inspection only and not tested on a real repo, so it's possible it doesn't work? Let's see if anyone else has tried it successfully

**Reply by viceice** [reply]
🫣 replacement value is wrong. sorry. must had a bad day. 🙈

**Reply by viceice** [reply]
will fix it now

**Reply by viceice** [reply, marked as answer]
- #22592

**Reply by fgreinacher** [reply]
Sorry for the late response, I was out on vacation. Seems we did not notice this because we already got the adaptions from the original PR (https://github.com/renovatebot/renovate/pull/20898).

Thanks for fixing @viceice!
